Makefile

```diff
@@ -6,6 +6,7 @@ MANAGER_IMG ?= ghcr.io/azure/eraser-manager:${VERSION}
 ERASER_IMG ?= ghcr.io/azure/eraser:${VERSION}
 COLLECTOR_IMG ?= ghcr.io/azure/collector:${VERSION}
 VULNERABLE_IMG ?= docker.io/library/alpine:3.7.3
+NGINX_IMG ?= docker.io/library/nginx:latest
```

Comment:
are we testing this for vulnerabilities? if so, this image will change all the time

Reply:
we are using this as the non-vulnerable image to remove when there is no scanner in scan disabled test

Reply:
this can have vulnerabilities though, can we create an image using `FROM scratch` perhaps?